packages/cli/src/commands/server/runServer.ts

```diff
@@ -9,13 +9,13 @@
 import Metro from 'metro';
```

Comment:
Can we also rename the directory from `server` to `start` now that most of the implementation moved to other package? I think it's a good moment to make it happen and be consistent with other commands

Reply:
Sure, renamed it to `start`.